docs/advanced-topics/wes/cli-wes-tutorial.rst

```diff
@@ -0,0 +1,148 @@
+the input JSON file, and any additional attachment files needed for your workflow.
+To launch a Dockstore workflow that takes an input JSON, you can run:
+
+   .. code:: bash
+
+        dockstore workflow wes launch --entry example/dockstore-entry/whalesay:master --json inputs.json
+
+GetRunStatus
+^^^^^^^^^^^^^^^^^^^^^^^^
+The official documentation for this endpoint can be found here: `GetRunStatus <https://ga4gh.github.io/workflow-execution-service-schemas/docs/#operation/GetRunStatus>`_
+
+This endpoint allows you to retrieve the run status of a launched workflow. After launching a WES workflow, a run ID is returned that can
+be used to query information regarding the run.
+To get the run status of a launched workflow, you can run:
+
+   .. code:: bash
+
+        dockstore workflow wes status --id example-runid-12345-67890
+
+GetRunLog
+^^^^^^^^^^^^^^^^^^^^^^^^
+The official documentation for this endpoint can be found here: `GetRunLog <https://ga4gh.github.io/workflow-execution-service-schemas/docs/#operation/GetRunLog>`_
+
+This endpoint allows you to retrieve the run logs of a launched workflow. After launching a WES workflow, a run ID is returned that can
+be used to query information regarding the run.
+To get the run log of a launched workflow, you can run:
+
+   .. code:: bash
+
+        dockstore workflow wes logs --id example-runid-12345-67890
+
+CancelRun
+^^^^^^^^^^^^^^^^^^^^^^^^
+The official documentation for this endpoint can be found here: `CancelRun <https://ga4gh.github.io/workflow-execution-service-schemas/docs/#operation/CancelRun>`_
+
+This endpoint allows you to cancel the execution of a launched workflow. After launching a WES workflow, a run ID is returned that can
+be used to cancel the workflow run.
+To cancel a launched workflow, you can run:
+
+   .. code:: bash
+
+        dockstore workflow wes cancel --id example-runid-12345-67890
+
+
+Configuring the Dockstore CLI
+------------------------------
+To configure the Dockstore CLI to make WES requests, the Dockstore CLI configuration file must be modified with a WES section. The configuration file is
+located by default in your home directory, at ``~/.dockstore/config``, although you can pass any config file to the Dockstore CLI using
+the ``--config`` parameter. The WES section of the config file specifies three values, the URL of the WES server, an authorization type, and
+an authorization value:
+
+*~/.dockstore/config*
+
+.. code:: text
+
+    [WES]
+    url: WES_URL
+    authorization: AUTHORIZATION_VALUE
+    type: AUTHORIZATION_TYPE
+
+The Dockstore CLI supports two authorization types, ``aws`` and ``bearer``:
+
+1. The ``aws`` authorization type indicates that the provided authorization value will be the string name of an AWS named profile, and that the Dockstore CLI must use the `AWS SigV4 <https://docs.aws.amazon.com/general/latest/gr/signature-version-4.html>`_ signing process for HTTP/HTTPS requests.
```

Comment:
AWS appears kinda suddenly here, maybe link to the other doc to say "this is why you might want to use AWS auth"

Reply:
Added a note: https://github.com/dockstore/dockstore-documentation/pull/173/commits/ce968c9a83207cc365ed10c61e980d05e90f97ee